src/southbridge/amd/pi/hudson/early_setup.c

```diff
@@ -207,6 +208,11 @@ static uintptr_t hudson_spibase(void)
 	return (uintptr_t)base;
 }
 
+uintptr_t lpc_get_spibase(void)
+{
+	return hudson_spibase();
+}
+
```

Comment:
I think it would be better to rename `hudson_spibase()` and make it globally visible (non-static) instead.

Reply:
Not part of this PR, it needs a rebase.